.circleci/main/@main.yml

```diff
@@ -565,7 +580,9 @@ workflows:
           matrix:
             parameters:
               architecture: [linux/amd64,linux/arm64,linux/arm/v7]
-          context: "docker-content-trust"
+          context:
+            - docker-content-trust
+            - Service Principal
```

Comment:
I would prefer a context name without a space.

Reply:
Service Principal context was there, it also is used on cloud repos so I didn't came up with the name.
Service Principal is the Azure name for a system user / account. 
I can change it to azure-cr-trust to make inline with the other context

Reply:
no specific opinion, really, tho' I agree that having a space in there feels weird...

if other things are relying on it, it's up to you whether you want to go through the trouble of renaming it everywhere else it's used